rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/41_knn_search_byte_quantized.yml

```diff
@@ -0,0 +1,342 @@
+setup:
+  - skip:
+      version: ' - 8.11.99'
+      reason: 'kNN float to byte quantization added in 8.12'
+  - do:
+      indices.create:
+        index: hnsw_byte_quantized
+        body:
+          mappings:
+            properties:
+              name:
+                type: keyword
+              vector:
+                type: dense_vector
+                dims: 5
+                index: true
+                similarity: l2_norm
+                index_options:
+                  type: hnsw
+                  quantization_options:
```

Comment:
Do we want to add any tests validating errors if we try to create quantized indices with unsupported values?